addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4831,7 +4831,31 @@ msgctxt "#2104"
 msgid "Check the log for more information."
 msgstr ""
 
-#empty strings from id 2105 to 2202
+#. Heading text for yes/no dialog to ask for add-on install to get his font file
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2105"
+msgid "Missing font language support"
+msgstr ""
+
+#. Used to show notification about automatic language add-on installation, for here the used text
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2106"
+msgid "Do you want to install the add-on to support characters?[CR][CR]Add-on: {0:s} ({1:s})[CR]Character Typ: \"{2:s}\""
+msgstr ""
+
+# Notification message if add-on installation about font failed.
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2107"
+msgid "Installation of the add-on \"{0:s}\" to support the written language \"{1:s}\" failed."
+msgstr ""
+
+#. Used to show notification about automatic language add-on installation, to report result by cancel.
+#: xbmc/guilib/GUIFontTTF.cpp
+msgctxt "#2108"
+msgid "Selected language add-on for specifying the font does not always correspond to the language used itself.[CR]If you choose "no", this install is interrupted in the currently running Kodi, but after a restart and the font is needed again, it is brought up again![CR]Are you sure to cancel?"
```

Comment:
This string crosses my github width twice over, seems like it would create a wall of text. Hint: people don't read. Try to give a few words that lets someone get through the dialog quickly.

Reply:
Have removed this dialog show and the text.